packages/elements/list/src/getListDeleteForward.ts

```diff
@@ -22,15 +30,52 @@ export const getListDeleteForward = (editor: SPEditor) => {
   const res = getListItemEntry(editor, {});
 
   let moved: boolean | undefined = false;
-  if (!isSelectionAtBlockEnd(editor) || !res) {
+  if (!isSelectionAtBlockEnd(editor)) {
     return moved;
   }
 
+  if (!res) {
+    const afterPoint = Editor.after(editor, editor.selection!.focus.path);
+
+    if (afterPoint) {
+      // there is a block after it
+      const nextSiblingListRes = getListItemEntry(editor, {
+        at: afterPoint.path,
```

Comment:
Any specific reason not to use the following?

```suggestion
        at: afterPoint,
```

Reply:
It does not cause any issue, I'll remove the .path in the next commit.